README.md

```diff
@@ -23,30 +28,34 @@ In the future, this tool is likely to evolve to support:
 ## How to Use
 
 ```bash
-cargo run --release -- --db-path <PATHFINDER_DB> --start-block <BLOCK_NUM> --end-block <BLOCK_NUM>
+cargo run --release -- --db-path <PATHFINDER_DB> --start-block <BLOCK_NUM> --end-block <BLOCK_NUM> [--svg-out <HISTOGRAM_FILENAME>]
 ```
 
 `PATHFINDER_DB` is the path of the Pathfinder sqlite database. The Pathfinder
 database is generated from running a `pathfinder` node. Further information is
 available
 [here](https://github.com/eqlabs/pathfinder/tree/v0.11.6?tab=readme-ov-file#database-snapshots).
 
+`HISTOGRAM_FILENAME` is the name of the file to output the libfunc histogram.
+The image is saved in SVG format.
+
 This tool makes use of `tracing` library for log purposes. For this reason set
 `RUST_LOG` at least at `info` level to see the raw output of libfunc statistics.
```

Comment:
I think this section should just give a happy use-case, rather than providing all the options! After all, people can run with `--help` to see the full list of flags they can pass! Focus on just what the MVP use-case is.